pages/index.tsx

```diff
@@ -14,7 +19,23 @@ const Home: NextPage = () => {
         <meta name="description" content="Generated by create next app" />
         <link rel="icon" href="/favicon.ico" />
       </Head>
-      <Layout>
+      <Layout hideHeaderSearch={true} marginY={"130px"}>
+        <Stack align={"center"}>
+          <HStack spacing={5} align={"center"}>
+            <Image
+              src={bookBazarLogo}
+              alt="Book Bazar Logo"
+              width={60}
+              height={60}
+            />
+            <Text fontSize="6xl" color="#017185" fontFamily="title">
```

Comment:
I think that we should change the colour of the logo text when the theme switches between light theme and dark theme, in order to match the behaviour of the buttons. In the dark theme, the teal buttons actually become a lighter variant of teal.

Here is what I did for my PR: https://github.com/calebmech/Book-Bazar/pull/106/files#diff-acfcca2cc15001c7c8e5abed2effd822d9327c02a9d2139885250d4cf23d4c24R29

Then again, the logo would also need to switch, which would probably be more hassle than its worth. Maybe we should lock in the theme to either the light theme or the dark theme, then go from there?

Reply:
Just set the `color` property on the parent to what David suggested and replace the PNG with this SVG.

![book-bazar-logo](https://user-images.githubusercontent.com/684772/152886035-4d5f5c2b-b3bf-4349-9807-092a7da7119c.svg)

Reply:
Just set the `color` property on the parent to what David suggested and replace the PNG with this SVG.

(You can try making the colour of the pages transparent instead of white if that looks better in dark mode too by changing the fill property of the second path in the SVG body)

![book-bazar-logo](https://user-images.githubusercontent.com/684772/152886749-df8af79a-ff17-46bc-82c7-ebc8389fc276.svg)